styles/styles.css

```diff
@@ -72,8 +71,10 @@ body {
   text-align: center;
 }
 
+
 .navigation {
   display: flex;
+  height: 64px;
```

Comment:
```suggestion
  min-height: 64px;
```
Фиксированное значение высоты в блоке с контентом может привести к таким последствиям:
![image](https://user-images.githubusercontent.com/17790771/201306397-f2b334ed-3421-479a-8fad-cd740cb75faf.png)